workflows/metatdenovo.nf

```diff
@@ -75,7 +75,6 @@ include { MEGAHIT_INTERLEAVED              } from '../modules/local/megahit/inte
 include { HMMRANK                          } from '../modules/local/hmmrank.nf'
 include { UNPIGZ as UNPIGZ_FASTA_PROTEIN   } from '../modules/local/unpigz.nf'
 include { UNPIGZ as UNPIGZ_CONTIGS         } from '../modules/local/unpigz.nf'
-include { FORMAT_TAX                       } from '../modules/local/format_tax.nf'
```

Comment:
Is this not needed anymore? If so, is the `format_tax.nf` deleted?

Reply:
It is now in subworkflow/local/eukulele.nf